src/Controller/Component/CategoriesComponent.php

```diff
@@ -45,6 +45,27 @@ public function index(?string $objectType = null, ?array $options = []): array
         return $apiClient->get('/model/categories', $options);
     }
 
+    /**
+     * Fetch categories names
+     *
+     * @param string|null $objectType The object type
+     * @return array
+     */
+    public function names(?string $objectType = null): array
+    {
+        $apiClient = ApiClientProvider::getApiClient();
+        $query = [
+            'fields' => 'name',
+            'page_size' => 1000,
```

Comment:
in BE4 API we have a hard `MAX_LIMIT` of 500 - i don't think `1000` will be ever used 

it's ok for now, but we should add more calls if there are more pages